shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/Arrival.scala

```diff
@@ -27,7 +27,17 @@ object Prediction {
   implicit val predictionInt: ReadWriter[Prediction[Int]] = macroRW
 }
 
-case class TotalPaxSource(pax: Option[Int], feedSource: FeedSource)
+case class Passengers(actual: Option[Int], transit: Option[Int]) {
+  def getPcpPax: Option[Int] = {
+    actual.map(_ - transit.getOrElse(0))
```

Comment:
We need logic in here to deal with the possibility of transit being higher than actual, which would result in negative pax. Also need a test around it